web/src/app/pages/rejectedList/rejectedList.controller.js

```diff
@@ -0,0 +1,85 @@
+(function() {
+    'use strict';
+
+    angular.module('BlurAdmin.pages.rejectedList')
+        .controller('RejectedCtrl', RejectedCtrl);
+
+    /** @ngInject */
+    function RejectedCtrl($scope, $http, $uibModal, toastr,internviewees) {
+
+        $scope.smartTablePageSize = 10;
+
+        $scope.dataInterviewee = [];
+
+        $scope.dataInterviewee = (internviewees);
+        $scope.Interviewee = [].concat($scope.dataInterviewee);
+
+        $scope.rejectedIntern = {};
+
+        var signUp = function(email, username, password) {
+            // cognito
+         //   AWSCognito.config.region = IG.cognitoConfigRegion;
+
+         //   var poolData = {
+        //        UserPoolId: IG.cognitoUserPoolId,
+        //        ClientId: IG.cognitoClientId
+        //    };
+
+        //    var userPool = new AWSCognito.CognitoIdentityServiceProvider.CognitoUserPool(poolData);
```

Comment:
Why is the cognito config is commented inside a page component. if they are not being used please remove the comments.